taskt/Core/Server/WebSocketClient.cs

```diff
@@ -185,10 +185,10 @@ private static void MessageReceived(object sender, MessageReceivedEventArgs e)
             SocketLogger.Information("Socket Message Received: " + e.Message);
 
             //server responded with script
-            if (e.Message.Contains("?xml"))
+            if (e.Message.StartsWith("{") && e.Message.EndsWith("}"))
```

Comment:
Server responses will largely be JSON I believe. This may cause false triggering on non-script responses.